lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
```

Comment:
@nden Why do you say that the following result is _wrong_?
```python
>>> LooseVersion('1.3.dev1555') > LooseVersion('1.3')
True # wrong
```
Is it because 'dev1555' corresponds to a version smaller than '1.3'?

Also, I do not think that you are using ``minversion`` correctly: each call to ``minversion`` returns a boolean (True/False). If your ``astropy`` version is '2.0' (like what I have on my machine) then both sides of your comparison statement return ``True`` and therefore ``True>True`` returns ``False``. That is the reason your ``minversion`` comparison
```python
>>> minversion(astropy,'1.3.dev1555') >minversion(astropy,'1.3')
False
```
returns False and not because it somehow miraculously can figure out that ``dev1555`` corresponds to a lower version than ``1.3`` (e.g., let's say ``dev1555`` corresponds to astropy version ``0.4.4``).

I think using ``LooseVersion`` is probably the most reliable way of handling this. For example, ``minversion`` was not even available in astropy in all versions. Finally, all ``minversion`` does is to use ``LooseVersion`` to do the same thing that I did in this PR.

I see no advantage in using ``minversion``.